clients/python/README.md

```diff
@@ -56,11 +56,40 @@ Owner and controller of cells in the game. As a programmer of an AI, you are a
             and a dumb AI will take over the player's cells until the player
             becomes active once again.
 
+### Cell
```

Comment:
Are you sure all that should be in the python doc? Shouldn't we start a global player doc?